_posts/14-02-01-Opcode-Cache.md

```diff
@@ -27,3 +28,4 @@ Read more about opcode caches:
 [Zend Optimizer+]: https://github.com/zendtech/ZendOptimizerPlus
 [WinCache]: https://www.iis.net/downloads/microsoft/wincache-extension
 [PHP_accelerators]: https://wikipedia.org/wiki/List_of_PHP_accelerators
+[PHP Preloading]: https://www.php.net/manual/fr/opcache.preloading.php
```

Comment:
Might be an idea to use the language agnostic link ? This should automatically load the page in the user preferred language.

See: https://www.php.net/urlhowto.php

```suggestion
[PHP Preloading]: https://www.php.net/opcache.preloading
```

Reply:
agreed, suggestion used :)

Reply:
great catch @jrfnl!

Reply:
Might be worth it to do a full review for links being language agnostic. I know I did one at some point, but that's quite some years ago by now.